<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-TPTMM17A" failures="1" tests="1" name="Testing Element - Edit" time="9.376" errors="0" timestamp="2022-09-26T21:07:17 IDT">
  <testcase classname="addressbook.EditElementTask" name="testViewTask" time="0.0">
    <failure type="java.lang.RuntimeException" message="java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 4">
      <![CDATA[java.lang.RuntimeException: java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 4
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 4
at addressbook.EditElementTask.viewTask(EditElementTask.java:51)
... 29 more
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- testViewTask -->
</testsuite> <!-- Testing Element - Edit -->
